plugin/storage/es/options_test.go

```diff
@@ -57,7 +57,7 @@ func TestOptionsWithFlags(t *testing.T) {
 		"--es.aux.server-urls=3.3.3.3, 4.4.4.4",
 		"--es.aux.max-span-age=24h",
 		"--es.aux.num-replicas=10",
-		"--es.tls=true",
```

Comment:
Breaking change! 

We should keep `--storage.tls=true`

Reply:
+1 we need to keep the existing option, but can be marked as deprecated - and from now on use `es.tls.enabled` as the preferred way to enable TLS.